R/post_ts.R

```diff
@@ -86,49 +88,19 @@ post_ts = function(files, on_exists=c("stop", "skip", "replace", "merge"), verbo
           dir.create(post_merge_dir, showWarnings=FALSE)
           files[i] <- write_ts(data=ts_merged, site=ts_path$site_name, var=ts_path$var, src=ts_path$src, folder=post_merge_dir)
           # delete the old one in preparation for overwriting
-          delete_ts(ts_path$var_src, ts_path$site_name, files_only=TRUE, verbose=verbose)
-          expect_id_loss <<- FALSE
+          item_rm_files(ts_id, files = basename(files[i]))
```

Comment:
in both 'merge' and 'replace', we could probably switch to [`item_replace_files`](https://github.com/USGS-R/sbtools/blob/master/R/item_replace_files.R), surgical version (`all=FALSE`), right? then put the `item_append_files` bit in a conditional that only runs if we need to add the new file? i guess it's the same calls to `item_rm_files` and `item_append_files` under the hood, but it seems like `item_replace_files` would be easier to read...thoughts?